python-sdk/pachyderm_sdk/client.py

```diff
@@ -353,8 +351,10 @@ def _create_channel(
     address: str,
     root_certs: Optional[bytes],
     options: MetadataType,
+    tls: bool = False,
```

Comment:
I don't love having both `root_certs` and `tls` as separate arguments, but I'm not sure of a better way to do this at the moment. Presumable if `root_certs!=None and tls=False` we still want to use the root certs in a secure channel? That appears to be the default behavior in the client

Reply:
yeah, i was also trying to find a way around having both but couldn't come up with a good way to do that. we essentially have 3 options we want to accommodate: 1) is insecure_channel via tls == false, 2) is secure_channel with default root_certs where tls == true and root_certs == None, and 3) is secure channel with specified root certs.